src/lib/components/ResponseMultipleChoice/index.js

```diff
@@ -95,7 +95,7 @@ class ResponseMultipleChoice extends BaseResponseElement {
                     @click=${this.onOptionClick}
                     class="web-select-group__input web-response-mc__input gc-analytics-event"
                     type="${this.selectType}"
-                    name="${this._formName}"
+                    name="web-response-mc-form-${this._formName}"
```

Comment:
I thought the salt function added this prefix too. Either way it's unique now.

Reply:
No, it doesn't. I kinda think it should though...